hive/src/main/java/org/apache/iceberg/hive/HiveTableOperations.java

```diff
@@ -65,18 +65,26 @@
 public class HiveTableOperations extends BaseMetastoreTableOperations {
   private static final Logger LOG = LoggerFactory.getLogger(HiveTableOperations.class);
 
+  private static final String HIVE_ACQUIRE_LOCK_STATE_TIMEOUT_MS = "hive.lock.acquire-timeout-ms";
```

Comment:
How about `iceberg.hive.lock-timeout-ms` instead? It isn't a Hive option, it is an Iceberg option for working with Hive; that's why I think we should use the `iceberg` namespace.